agent/xds/rbac_test.go

```diff
@@ -552,72 +556,260 @@ func TestMakeRBACNetworkAndHTTPFilters(t *testing.T) {
 		}
 	)
 
+	makeL4Spiffe := func(name string, entMeta *acl.EnterpriseMeta) string {
```

Comment:
These tests are a bit different from the other v2 xds tests because we are using completely different code in v1 and v2. The tests are colocated and sharing the same golden files so we can visually inspect the v1 and v2 intentions / traffic permissions to show that they are equivalent.